### DISCUSSION TITLE
Option to run `cargo update` after bumping dependencies

### DISCUSSION THREAD

**Discussion by torokati44** [renovatebot/renovate#23960] [Suggest an Idea]
### Tell us more.

This got moved here from https://github.com/renovatebot/renovate/discussions/23958.

It would be useful for us if the bot ran `cargo update` before submitting its PRs to make sure `Cargo.lock` is entirely fresh, including transitive dependencies.

**Comment by rarkins** [comment]
Created feature request #23971
